source/renderer/app/stores/WalletSettingsStore.js

```diff
@@ -79,6 +82,24 @@ export default class WalletSettingsStore extends Store {
   getWalletsRecoveryPhraseVerificationData = (walletId: string) =>
     this.walletsRecoveryPhraseVerificationData[walletId] || {};
 
+  getWalletShowUsedAddressesStatuses = (walletId: string) =>
+    this.walletsShowUsedAddressesStatuses[walletId] || {};
+
+  @computed get walletsShowUsedAddressesStatuses() {
+    const { all: walletsLocalData } = this.stores.walletsLocal;
+    // $FlowFixMe
```

Comment:
@aleksandardjordjeviciohk could you check this method? It could be much simpler, with another fn `getWalletSettingsById`, then returning `showUsedAddresses`.

Could you also check the `walletsRecoveryPhraseVerificationData` method, as it also uses `$FlowFixMe`?

Reply:
I mean `getLocalWalletDataById`.

Reply:
Fixed